tests/extmod/select_poll_fd.py

```diff
@@ -36,9 +36,14 @@
 
 # Test registering a very large number of file descriptors.
 poller = select.poll()
-for fd in range(6000):
+for fd in range(1, 1000):
```

Comment:
This needs to be 6000 to hit the OSError in the test below.

Reply:
If it needs to be 1000 so the final test in this file succeeds, then I suggest creating a new poller object which just goes up to 1000, and keeping this particular test as-is.

Reply:
Fixed.